chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/DeveloperManagerImpl.java

```diff
@@ -780,6 +795,33 @@ public String toString() {
         }
     }
 
+    private static boolean isTransparencyAttestationUpdated(final DeveloperDTO original, final DeveloperDTO changed) {
+        if ((original.getTransparencyAttestationMappings() != null && changed.getTransparencyAttestationMappings() == null)
+                || (original.getTransparencyAttestationMappings() == null && changed.getTransparencyAttestationMappings() != null)
+                || (original.getTransparencyAttestationMappings().size() != changed.getTransparencyAttestationMappings().size())) {
+            return true;
+        } else {
+            for (DeveloperACBMapDTO originalMapping : original.getTransparencyAttestationMappings()) {
+                for (DeveloperACBMapDTO changedMapping : changed.getTransparencyAttestationMappings()) {
+                    if (isBasicPropertyChanged(originalMapping.getAcbName(), changedMapping.getAcbName())
+                            || isBasicPropertyChanged(originalMapping.getTransparencyAttestation(),
+                                    changedMapping.getTransparencyAttestation())) {
+                        return true;
+                    }
+                }
+            }
+        }
+        return false;
+    }
+
+    private static boolean isBasicPropertyChanged(final Object originalProperty, final Object otherProperty) {
```

Comment:
There is an ObjectUtils.equals or maybe it's isEqual method in I think either a Spring or Apache library and I wonder if that does something similar to this and you could use it instead? If not, may want to consider moving this method into a Utils class of our own as it seems like a utility

Reply:
I will do one of those two things.

Reply:
Thanks - I ended up using the JDK7 Objects.equals method for this as you were likely referencing. There is a Spring one too called nullSafeEquals that is the same but can't really beat using the JDK from a dependency standpoint. All I had to do was reverse the result since I was checking for a change not equality.